eras/babbage/test-suite/cardano-ledger-babbage-test.cabal

```
@@ -32,16 +32,16 @@ library
 
     build-depends:
         base >=4.14 && <4.19,
-        cardano-ledger-allegra >=1.1,
+        cardano-ledger-allegra >=1.3,
         cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
-        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
-        cardano-ledger-alonzo-test >=1.1,
-        cardano-ledger-babbage:{cardano-ledger-babbage, testlib} ^>=1.5,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.8.1,
-        cardano-ledger-shelley-ma-test >=1.1,
-        cardano-ledger-mary >=1.4,
-        cardano-ledger-shelley-test >=1.1,
-        cardano-ledger-shelley >=1.6 && <1.9,
+        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.6,
+        cardano-ledger-alonzo-test >=1.2,
+        cardano-ledger-babbage:{cardano-ledger-babbage, testlib} ^>=1.6,
+        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.10,
+        cardano-ledger-shelley-ma-test >=1.3,
+        cardano-ledger-mary >=1.5,
+        cardano-ledger-shelley-test >=1.4,
+        cardano-ledger-shelley >=1.9 && <1.10,
```

Because nothing has changed in this package we should be touching the lower bound, only relaxing the upper bound:
```suggestion
        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
        cardano-ledger-alonzo-test >=1.1,
        cardano-ledger-babbage:{cardano-ledger-babbage, testlib} >=1.5 && <1.7,
        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.8.1,
        cardano-ledger-shelley-ma-test >=1.1,
        cardano-ledger-mary >=1.4,
        cardano-ledger-shelley-test >=1.1,
        cardano-ledger-shelley >=1.6 && <1.10,
```